### ISSUE TITLE
How can I download the data?

### ISSUE THREAD

**Issue by drogozhang** [gary1346aa/Fast-RCNN-Pytorch#1]
.pkl file in the code.
I am not sure about its structure
So plz give a link at least.
thanks a lot.

**Comment by zk31601102** [comment]
I found that the network for cls_score did not use the softmax. is that a bug or special implement?

**Comment by drogozhang** [comment]
Ok I know that.

**Comment by lixiaoming0017** [comment]
> Ok I know that.

please give a link to download the train_data.pkl

**Comment by drogozhang** [comment]
> > Ok I know that.
> 
> please give a link to download the train_data.pkl

Sorry. I search the key word of file name and I think the dataset should for ImageNet Object Detection.  And It's very very huge. Original tar is more than 30 GB. So I just know the dataset. (You can see the file name in jupyter notebook file.

BTW, the author seems like not running his github anymore. Without reply after I try to email him several  months ago.

**Comment by gary1346aa** [comment]
https://www.kaggle.com/c/datalabcup-cnn-object-detection/data

**Comment by VanHongNguyen** [comment]
> https://www.kaggle.com/c/datalabcup-cnn-object-detection/data

Can you give me another link to download the data ,because that competition no more accept participation so we have no access to that data,This is a limited-participation competition. Only invited users may participate.
